internal/dinosaur/pkg/services/dinosaur.go

```diff
@@ -54,9 +63,9 @@ type DinosaurService interface {
 	HasAvailableCapacity() (bool, *errors.ServiceError)
 	// HasAvailableCapacityInRegion checks if there is capacity in the clusters for a given region
 	HasAvailableCapacityInRegion(dinosaurRequest *dbapi.CentralRequest) (bool, *errors.ServiceError)
-	// PrepareDinosaurRequest sets any required information (i.e. dinosaur host, sso client id and secret)
-	// to the Dinosaur Request record in the database. The dinosaur request will also be updated with an updated_at
-	// timestamp and the corresponding cluster identifier.
+	// AcceptDinosaurRequest transitions CentralRequest to 'Preparing'.
+	AcceptDinosaurRequest(dinosaurRequest *dbapi.CentralRequest) *errors.ServiceError
```

Comment:
```suggestion
	// AcceptCentralRequest transitions CentralRequest to 'Preparing'.
	AcceptCentralRequest(centralRequest *dbapi.CentralRequest) *errors.ServiceError
```
Let's keep the dinosaur count monotonically decreasing

Reply:
Hehe :). Yeah, I was on the fence here. I didn't spawn any dinosaurs in the newly created file, but here I'm not sure which strategy makes most sense: being consistent and 🪓 all dinosaurs at once or use the methodical one-dinosaur-at-a-time approach.